types/wlr_output_layout.c

```diff
@@ -339,8 +339,12 @@ void wlr_output_layout_closest_point(struct wlr_output_layout *layout,
 		}
 	}
 
-	*dest_x = min_x;
-	*dest_y = min_y;
+	if (dest_x) {
+		*dest_x = min_x;
+	}
+	if (dest_y) {
+		*dest_y = min_y;
+	}
```

Comment:
We can return early from this function without doing anything if these are both NULL, right?

Reply:
Fixed. We should probably do more NULL checks like this for out variables.